build_tools/bazel/deep_copy.bzl

```diff
@@ -43,6 +43,7 @@ def _deep_copy_recursion_depth_1(x):
 
 def deep_copy(x):
     """Returns a copy of the argument, making a deep copy if it is a container.
+
```

Comment:
Uh... this has nothing to do with the change. I apparently just fixed a thing buildifier was complaining about. I can revert